### DISCUSSION TITLE
How to add module on website running?

### DISCUSSION THREAD

**Discussion by natthawatsuwan** [OrchardCMS/OrchardCore#9356] [Q&A] [answered]
Now, my Orchard running on Docker.
I want to add custom module to my Orchard without re-build image.
how to add ?
try deployment feature, but not work

**Comment by EricBui619** [comment, marked as answer]
@natthawatsuwan  I think it can't. As you know the image is built from your code -> container is built based on image. So whenever there are some changes in your code, docker  image and container needs to be rebuilt to update new feature. For reference, you can read this article: https://stackoverflow.com/questions/52689570/how-to-change-source-code-without-rebuilding-image-in-docker
